core/src/main/java/ch/cyberduck/core/CredentialManagerPasswordStoreDescriptorService.java

```diff
@@ -0,0 +1,65 @@
+package ch.cyberduck.core;
+
+/*
+ * Copyright (c) 2002-2023 iterate GmbH. All rights reserved.
+ * https://cyberduck.io/
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.net.URI;
+
+/**
+ * Implementation for Windows Credentials Manager
+ */
+public class CredentialManagerPasswordStoreDescriptorService extends DefaultPasswordStoreDescriptorService {
+
+    @Override
+    public String getDescriptor(final Host bookmark) {
+        return bookmark.getProtocol().getIdentifier();
```

Comment:
Using the identifier will not allow sharing between different protocols.

Reply:
The requirement is only that this behaviour can be overridden by a different implementation.